github-issue-summary-to-email/main.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/github;
+import wso2/choreo.sendemail as email;
+
+// Github client configuration parameters
+configurable http:BearerTokenConfig gitHubTokenConfig = ?;
+configurable string repositoryName = ?;
+configurable string repositoryOwner = ?;
+
+// Email configuration parameters 
+configurable string recipientAddress = ?;
+
+public function main() returns error? {
+    string assigneeSummary = "";
+    github:Client githubClient = check new ({
+        auth: {
+            token: gitHubTokenConfig.token
+        }
+    });
+    stream<github:User,github:Error?> collaborators = check githubClient->getCollaborators(repositoryOwner, repositoryName);
+    log:printInfo("Started compiling the report");
```

Comment:
Unnecessary log line

Reply:
Updated in [5131705](https://github.com/ballerina-guides/integration-samples/pull/35/commits/51317055927a041c645786b20bcb6af3864f1081)